docs/src/main/resources/news/2018-07-26-first-release.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: news
+title: First release
+date: 2018-07-26 08:00:00
+section: news
+author:
+name: Jeroen Laverman
+id: laverman
+---
+{% include base.html %}
+First release available on <a href="https://mvnrepository.com/artifact/org.eclipse.hawkbit">Maven Central</a>
+
+# Eclipse hawkBit 0.2.0 - First Release
+
+hawkBit is a domain independent back-end framework for rolling out software updates to constrained edge devices as well 
+as more powerful controllers and gateways connected to IP based networking infrastructure. It is part of the Eclipse IoT 
+since 2015 and with _0.2.0_ a first release is available.
+
+In this article, we want to give an overview over the latest highlights of hawkBit and let you know how you can get 
+tarted in seconds.
+
+## Finally, it is here! 
+
+After being around in the Eclipse IoT realm for quite some time now, we are more than happy to announce our first release: 
+_Eclipse hawkBit 0.2.0_. The release can be found on [Maven Central](https://mvnrepository.com/artifact/org.eclipse.hawkbit) 
+and [Docker Hub](https://hub.docker.com/r/hawkbit/hawkbit-update-server/). It includes the following core features:
+
+* Device and Software Repository
+* Update Management
+* Artifact Content Delivery
+* Rollout Management
+
+which are accessible via the following interfaces:
+
+* Management UI
+* Management API
+* Direct Device Integration (DDI) API
+* Device Management Federation (DMF) API
+
+![hawkBit Overview](hawkBit_overview.jpeg)
+
+
+## What's new?
```

Comment:
Don't forget to mentioned that we finally open sourced the rest docs: https://github.com/eclipse/hawkbit/pull/688

Reply:
Good point!